x-pack/plugin/deprecation/src/test/java/org/elasticsearch/xpack/deprecation/NodeDeprecationChecksTests.java

```diff
@@ -42,17 +43,19 @@
 public class NodeDeprecationChecksTests extends ESTestCase {
 
     public void testCheckDefaults() {
-        final Settings settings = Settings.EMPTY;
+        final Settings settings = Settings.builder().put(XPackSettings.SECURITY_ENABLED.getKey(), true).build();
```

Comment:
The method is `Defaults`, but that's not the default setting.
I don't think the method should lie about what it does. Either we:
1. Rename it
2. Change the license state to be gold+ (which is still not "default", but better)
3. Accept that the default now has a deprecation.

I think we want (3)

Reply:
I was in "mechanically apply change" mode I guess